README.md

```diff
@@ -71,10 +71,28 @@ packages/engines/dashboards-engine/addon/templates/default.hbs
 ...
 ```
 
-### Folder namespace 
-If your source entities are namespaced within a folder you can use the `-f` option to specify the same 
-before copying. 
-Say for example you want to move a component called `widget` from `app/components/dashboards` 
+### Keep-tests-in-host
```

Comment:
Can we rename this option as Skip-tests and have the arguments in the form
`--skip-tests` and `-st`